styx-api-service/src/main/java/com/spotify/styx/api/BackfillResource.java

```diff
@@ -164,12 +165,13 @@ private BackfillsPayload getBackfills(RequestContext rc) {
     final Optional<String> workflowOpt = rc.request().parameter("workflow");
     final boolean includeStatuses = rc.request().parameter("status").orElse("false").equals("true");
     final boolean showAll = rc.request().parameter("showAll").orElse("false").equals("true");
+    final Instant start = rc.request().parameter("start").map(ParameterUtil::parseDate).orElse(Instant.MIN);
```

Comment:
Can you add a test that shows that we pick the value from the request and also that we use Instant.Min when it is not in the request